### DISCUSSION TITLE
Implement an 'outbox', like pretalx

### DISCUSSION THREAD

**Discussion by glasnt** [pretix/pretix#2148] [Ideas]
Hello!

This is a feature request, not a bug and not an active issue. I would like to have the same verify and send outbox functionality that pretalx has in pretix.

User story: I batch send emails and I want to verify the templating, etc, before sending many emails. If I send emails as vouchers, I want to preview the emails before they go out, and have a history of emails that go out.

**Comment by raphaelm** [comment]
Hey, thanks a lot :)

First of all, I like the outbox concept a lot. It is especially useful in pretalx due to the submission review / acceptance process where it sometimes is really useful to have an "undo" option.

In pretix *almost* all emails are either sent fully automatically and should stay that way (e.g. order confirmation) *or* already have a "preview" button (e.g. the mass email feature, the new automated emails). Also, all emails related to an order (90% of emails) have an easily accessible history to look at.

The "sending vouchers" feature is currently the only exception I can think of that doesn't match this. Therefore, I'm unsure if its worthwile making the user interface more complicated by adding an outbox concept that requires you to do more clicks e.g. for a custom email (we already introduce more clicks and safeguards in #2099).

Maybe we should rather find a solution for the specific voucher use case as well?
